### DISCUSSION TITLE
English adjectives with dep=amod incorrectly tagged as verbs.

### DISCUSSION THREAD

**Discussion by rhdunn** [explosion/spaCy#13054] [Help: Other Questions]
## How to reproduce the behaviour
Given:
```python
import spacy

# FORM LEMMA UPOS FEATS DEPREL
def format(text):
    nlp = spacy.load("en_core_web_sm")
    for sent in nlp(text).sents:
        print(f"# text = {sent.text}")
        for tok in sent:
            feats = '_' if len(tok.morph) == 0 else str(tok.morph)
            print(f"{tok.text}\t{tok.lemma_}\t{tok.pos_}\t{feats}\t{tok.dep_}")
        print()

format("The haunted house down the street.")
format("It was the haunted house down the street.")
```

That outputs the following results:
```
# text = The haunted house down the street.
The     the     DET     Definite=Def|PronType=Art       det
haunted haunted ADJ     Degree=Pos      amod
house   house   NOUN    Number=Sing     ROOT
down    down    ADP     _       prep
the     the     DET     Definite=Def|PronType=Art       det
street  street  NOUN    Number=Sing     pobj
.       .       PUNCT   PunctType=Peri  punct

# text = It was the haunted house down the street.
It      it      PRON    Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs  nsubj
was     be      AUX     Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin   ROOT
the     the     DET     Definite=Def|PronType=Art       det
haunted haunt   VERB    Aspect=Perf|Tense=Past|VerbForm=Part    amod
house   house   NOUN    Number=Sing     attr
down    down    ADP     _       prep
the     the     DET     Definite=Def|PronType=Art       det
street  street  NOUN    Number=Sing     pobj
.       .       PUNCT   PunctType=Peri  punct
```

In both cases, "haunted" is annotated correctly as `dep=amod`, however it is incorrectly assigned `VERB/VBP` instead of `ADJ/JJ` in the second example. This is causing the LEMMA to be incorrect as well, due to the lemmatization following the rules for verbs.

There are other instances where this happens, so it is not just limited to this specific sentence structure or adjective.

## Your Environment
- **Operating System:** Windows 11
- **spaCy version:** 3.5.3
- **Platform:** MINGW64_NT-10.0-22621-3.4.6.x86_64-x86_64-64bit
- **Python version:** 3.11.2
- **Pipelines:** en_core_web_lg (3.5.0), en_core_web_md (3.5.0), en_core_web_sm (3.5.0)

**Comment by rmitsch** [comment]
Hi @rhdunn, thanks for reporting this! spaCy's has separate models for the parser and tagger components - they both can make mistakes and disagree. If this is a critical issue for you, you could consider post-processing.
